cnd/src/db/new_types.rs

```diff
@@ -0,0 +1,44 @@
+use ethereum_support::{FromDecimalStr, U256};
+use std::{fmt, str::FromStr};
+
+#[derive(Debug, Clone, Copy, PartialEq, derive_more::FromStr, derive_more::Display)]
+pub struct Satoshis(pub u64);
```

Comment:
:wrench: I think the correct plural is
```suggestion
pub struct Satoshi(pub u64);
```

Reply:
Highly likely to be true! Can I blatantly also ask for a follow-up PR here (all can go in one ofc!) :grimacing: